lib/components/Modal/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./Bottom";
```

Comment:
```suggestion
export * from "./bottom";
```
as per @jia-deriv 's suggestion

Reply:
Will rename